### PR TITLE
ZA2-Kwanda Silekwa-JavaScript-Core-1-Coursework-Week1

### PR DESCRIPTION
**Pull request by kwanda9700**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Kwanda Silekwa
- Your City: Cape town
- Your Slack Name: Kwanda Silekwa

# Homework Details

- Module: JS Core 1
- Week: 1

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

**Review by Riyaazct — COMMENTED** [review]
Hi Kwanda. you finished the Extra exercises as well, Amazing bro. well done.

**Comment by kwanda9700** [comment]
Thank you Riyaaz

**Comment by PatouMabando** [comment]
Hello Kwanda, I like the way you follow the steps to make your code more understandable. Well done.

File: exercises/C-variables/exercise.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `greeting`
+let greeting = "Hello World";
 
-console.log(greeting);
+for (i = 1; i < 4; i++) {
```
Comment (dschouw-CYF) [inline exercises/C-variables/exercise.js:4]: Great use of a for loop to perform a repetitive task 👍

File: exercises/L-functions-nested/exercise.js [hunk 2]
```diff
@@ -1,5 +1,29 @@
+function shoutyGreeting(str) {
```
Comment (dschouw-CYF) [inline exercises/L-functions-nested/exercise.js:1]: The exercise requires two functions, one that spells their names in  uppercase, and one that create a shouty greeting.
While you code works perfectly, maybe consider 
```suggestion
function shoutyGreeting(str) {  // logs the output ( shout greeting )
.....
function upperCaseName(str) { // uppercases the
```

**Review by dschouw-CYF — COMMENTED** [review]
Awesome work Kwanda!

**Pull request closed without merging**